packages/help-center/src/components/help-center-search-results.tsx

```diff
@@ -103,24 +104,34 @@ function HelpSearchResults( {
 }: HelpSearchResults ) {
 	const dispatch = useDispatch();
 
-	const hasPurchases = useSelector( hasCancelableUserPurchases );
-	const sectionName = useSelector( getSectionName );
+	const { hasPurchases, sectionName, adminResults, isSimpleSite } = useSelector( ( state ) => {
+		return {
+			hasPurchases: hasCancelableUserPurchases( state ),
+			sectionName: getSectionName( state ),
+			adminResults: getAdminHelpResults( state, searchQuery, 3 ),
+			isSimpleSite: getIsSimpleSite( state ),
+		};
+	} );
+
 	const isPurchasesSection = [ 'purchases', 'site-purchases' ].includes( sectionName );
 	const siteIntent = useSiteOption( 'site_intent' );
 	const rawContextualResults = useMemo(
 		() => getContextResults( sectionName, siteIntent ),
 		[ sectionName, siteIntent ]
 	);
 	const locale = useLocale();
-	const adminResults = useSelector( ( state ) => getAdminHelpResults( state, searchQuery, 3 ) );
 	const contextualResults = rawContextualResults.filter(
 		// Unless searching with Inline Help or on the Purchases section, hide the
 		// "Managing Purchases" documentation link for users who have not made a purchase.
 		filterManagePurchaseLink( hasPurchases, isPurchasesSection )
 	);
-	const { data: searchData, isLoading: isSearching } = useHelpSearchQuery( searchQuery, locale );
+	const { data: searchData, isLoading: isSearching } = useHelpSearchQuery(
+		searchQuery,
+		locale,
+		isSimpleSite
+	);
 
-	const searchResults = searchData?.wordpress_support_links ?? [];
+	const searchResults = searchData ?? [];
```

Comment:
Same as above. You removed the `wordpress_support_links` from here, but the API endpoint in our plugin still returns this in an array.